### DISCUSSION TITLE
Is indent still experimental?

### DISCUSSION THREAD

**Discussion by gwww** [nvim-treesitter/nvim-treesitter#3162] [Q&A] [answered]
Is it usable for any languages? I did a quick experiment with Lua and it seemed to work but I understand there's much more to it than my simple experiment.

**Comment by theHamsta** [comment, marked as answer]
I'd say yes: indent is still experimental. We have a lot of open issues for some issues, but that also has to do with the parsers of the languages. Some of them are very "indent"-friendly while we still struggle especially with Python (like most indent-significant languages) when there are error in the syntax tree and error recovery doesn't work well.

**Reply by Decodetalkers** [reply]
> I'd say yes: indent is still experimental. We have a lot of open issues for some issues, but that also has to do with the parsers of the languages. Some of them are very "indent"-friendly while we still struggle especially with Python (like most indent-significant languages) when there are error in the syntax tree and error recovery doesn't work well.

What is the function of indent about? Will it define how indent happened?
